appendices/spirvenv.adoc

```diff
@@ -425,18 +425,10 @@ endif::VK_VERSION_1_1[]
   * [[VUID-{refpage}-OpGroupNonUniformBallotBitCount-04685]]
     If code:OpGroupNonUniformBallotBitCount is used, the group operation
     must: be limited to *Reduce*, *InclusiveScan*, or *ExclusiveScan*
-ifndef::VK_EXT_mesh_shader[]
   * [[VUID-{refpage}-None-04686]]
```

Comment:
no need to change the VUID as it has already been updated to include `TaskPayloadWorkgroupEXT`

https://github.com/KhronosGroup/SPIRV-Tools/commit/4c456f7da67c5437a6fb7d4d20d78e2a5ae2acf2#diff-38fd3032d90cb56b9e25b2b4141235b736de1ad8efc70d02dabf2be8adaf0117R211